ngraph/core/src/op/quantize.cpp

```diff
@@ -161,3 +163,53 @@ shared_ptr<Node> op::Quantize::clone_with_new_inputs(const OutputVector& new_arg
     return make_shared<Quantize>(
         new_args.at(0), new_args.at(1), new_args.at(2), m_type, m_axes, m_round_mode);
 }
+
+bool op::Quantize::evaluate(const HostTensorVector& output_values,
```

Comment:
As I know at the current moment we don't have a networks where we have a `Quantize` operation.
I think we don't need to add this `evaliate` method.

Reply:
Right, I've removed the evaluator and restored the code in the interpreter backend for this op.

Reply:
Hmm, I have a general question do we have tests which use this op? Or maybe we can remove a reference implementation and a code from interpreter backend.

Reply:
There's quite a lot of tests of the ONNX Quantize operator that rely on the reference implementation `ngraph/test/backend/quantize_dequantize.in.cpp`. They are now running on INTERPRETER only because the CPU plugin does not support the Quantize operation. If we remove the code, we loose all tests for this op.
Perhaps we should address it in a separate PR along with onnx_importer changes so that it doesn't produce the Quantize op any more?

Reply:
Yes agree.